.github/workflows/main.yml

```diff
@@ -32,3 +31,11 @@ jobs:
     # Runs a single command using the runners shell
     - name: Compile and run tests on robot code
       run: ./gradlew build
+
+    # Runs a simulation to spot check crashes
+    - name: Run simulation to check for runtime errors
+      run: timeout 1m ./gradlew simulateJava 2>&1 | awk -v rc=0 '/The robot program quit unexpectedly/ { rc=1 } 1; END {exit rc}'
```

Comment:
If I run the command above without Robot errors, then the return code is 143. ( SIGTERM sent )

seems that awk is receiving the sigterm from timeout?

Reply:
It seems to work in GitHub's linux environment (CI pipeline is clean). Where are you running the command?

Reply:
Running Linux extensions for windows.  There might be subtle difference in signaling.  will go ahead and close this if it works in github's linux